src/sentry/runner/commands/configoptions.py

```diff
@@ -273,11 +273,18 @@ def sync(ctx: click.Context) -> None:
                                 )
                                 presenter_delegator.flush()
                                 raise
-
                         presenter_delegator.unset(opt.name)
                     else:
-                        presenter_delegator.drift(opt.name, "")
-                        drift_found = True
+                        if (
+                            options.can_update(
```

Comment:
if an option is defined in the automator file, and it is set on disk, we check for it here https://github.com/getsentry/sentry/blob/fix/options-on-disk/src/sentry/options/manager.py#L188-L200

otherwise, if it is set on disk but not defined in the automator, since options.get() and options.isset() will still return true,(even if the options DNE in the DB) we again check to see if the option is on disk. if it is, we continue.